servant/servant.cabal

```diff
@@ -1,5 +1,5 @@
 name:                servant
-version:             0.8
+version:             0.8.1
```

Comment:
It'll actually have to be a major version bump, but don't do that in this PR. Leave it '0.8' and we'll bump the versions when releasing.